test/Error/aggregate_error.js

```diff
@@ -0,0 +1,384 @@
+//-------------------------------------------------------------------------------------------------------
+// Copyright (C) Microsoft Corporation and contributors. All rights reserved.
+// Copyright (c) 2021 ChakraCore Project Contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
+//-------------------------------------------------------------------------------------------------------
+
+WScript.LoadScriptFile("..\\UnitTestFramework\\UnitTestFramework.js");
+
+var tests = [
+   {
+       name: "proptotype and constructor",
+       body: function ()
+       {
+            assert.areEqual(AggregateError, AggregateError.prototype.constructor);
+            assert.isFalse(AggregateError.prototype.hasOwnProperty("errors"));
+            assert.areEqual("", AggregateError.prototype.message);
+            assert.areEqual("AggregateError", AggregateError.prototype.name);
+            assert.areEqual("object", typeof AggregateError.prototype);
+            assert.areEqual(Error.prototype, Object.getPrototypeOf(AggregateError.prototype));
+            assert.areEqual("AggregateError", AggregateError.name);
+            assert.areEqual(2, AggregateError.length);
+        }
+    },
+    {
+        name: "errors iterable to list failures",
+        body: function ()
+        {
+            class TestError extends Error {}

```

Comment:
Can we neaten this up?

Name should be something like "AggregateError invalid iterable cases"

And then please can we neaten up the cases below - in most cases the assertion statements can be single line to make this file a lot shorter + comment on what they're doing:
 - the first case is "Iterator method that throws"
 - the second is "Iterator with no next() method"
 ...